emukit/bayesian_optimization/acquisitions/expected_improvement_per_cost.py

```diff
@@ -0,0 +1,61 @@
+from typing import Tuple, Union
+
+import numpy as np
+
+from ...core.interfaces import IModel, IDifferentiable
+from ...core.acquisition import Acquisition
+from emukit.bayesian_optimization.acquisitions import ExpectedImprovement
+
+
+class ExpectedImprovementPerCost(Acquisition):
+
+    def __init__(self, model: Union[IModel, IDifferentiable], cost_model: Union[IModel, IDifferentiable],
+                 jitter: np.float64 = np.float64(0))-> None:
+        """
+        This acquisition computes for a given input the improvement over the current best observed value in
```

Comment:
the information that it is per cost is missing. Also as of now i believe the cost function can be evaluated exactly? if not there is a distinction in between expectation of ei/cost and expectation ei/expectation cost. this class computes the latter in the more general case.

Reply:
you're right I will make that more clear